oauthproxy_test.go

```diff
@@ -986,3 +986,47 @@ func TestAjaxForbiddendRequest(t *testing.T) {
 	mime := rh.Get("Content-Type")
 	assert.NotEqual(t, applicationJSON, mime)
 }
+
+func TestClearSplittedCookie(t *testing.T) {
```

Comment:
Nit: Could we rename this to `TestClearSplitCookie`?

Reply:
done